Tasks/Loktionova/Task1/task1.py

```diff
@@ -0,0 +1,5 @@
+sum = 20000
+rate = 0.15
+term = 60
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice